test/tf/tf_test.cpp

```diff
@@ -706,6 +706,32 @@ TEST_CASE(relu6_test)
     EXPECT(p == prog);
 }
 
+TEST_CASE(relu6_mismatch_test)
+{
+    migraphx::program p;
+
+    auto* mm = p.get_main_module();
+    std::vector<size_t> input_lens{1, 3, 13, 37};
+    auto l0      = mm->add_parameter("0", migraphx::shape{migraphx::shape::half_type, input_lens});
+    auto min_val = mm->add_literal(0.0f);
+    auto max_val = mm->add_literal(6.0f);
+
+    auto l0_convert = mm->add_instruction(migraphx::make_op("convert",
+                                        {{"target_type", migraphx::to_value(migraphx::shape::float_type)}}), 
```

Comment:
I think you can just keep "target_type" to `migraphx::shape::float_type` instead of `to_value(float_type)`.